packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -29,10 +29,10 @@
 
 // Patterns - Blocks
 @import 'patterns/blocks/contentgroupcards/contentgroupcards';
+@import 'patterns/blocks/pictogramarray/content-group-pictograms';
```

Comment:
Shouldn't be called `pictogramarray` anymore

Reply:
I've changed this in the latest commit, Thanks!